internal/landing/html/inspect.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Inspect debug data </title>
+    </head>
+    <body>
+        <form class="form-group" action="/inspect" method="POST">
+            <div class="form-group">
+                <label for="cluster-id">Enter Cluster ID :</label>
+                <input type="text" id="cluster-id" name="cluster-id" class = "form-control" placeholder="sjdhfUG@%HKA^Srasfjskfj9987sfkj"><br>    
+            </div>   
+            <button type="submit" class="btn btn-default">Submit</button>
+        </form>
+        <br><hr>
+        <h4>The affiliates associated with given cluster-id {{.ClusterId}}: </h4>
+        {{if not .Affiliates}}
+            <p>No Affiliate data found</p>
+        {{end}}
+        <ol>
+            {{range .Affiliates}} 
+                <li class="affiliate-id">Affliate ID: {{.ID}} </li>
+                <ul>
+                    <li class="affiliate-data">Affliate Data: {{.Data}} </li>
```

Comment:
`.Data` is `[]byte`, we can use template function to format it other way. 

not tested, but something like this might work: `<code>{{ .Data | printf "%x" }}</code>`